src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppfølgingsoppgaveBeskrivelse.kt

```diff
@@ -9,21 +9,21 @@ import java.util.Locale
 object OppfølgingsoppgaveBeskrivelse {
 
     fun beskrivelseFørstegangsbehandlingInnvilget(periode: Pair<LocalDate, LocalDate>, vedtak: Vedtaksperiode): String {
-        return "Overgangsstønad er innvilget fra ${periode.vedtaksPeriodeToString()} " +
-                "Aktivitet: ${vedtak.aktivitet.name.enumToReadable()}"
+        return "Overgangsstønad er innvilget fra ${periode.vedtaksPeriodeToString()}. " +
+                "Aktivitet: ${vedtak.aktivitet.name.enumToReadable()}."
     }
 
     fun beskrivelseFørstegangsbehandlingAvslått(vedtaksdato: LocalDate): String {
-        return "Søknad om overgangsstønad er avslått i vedtak datert ${vedtaksdato.toReadable()}"
+        return "Søknad om overgangsstønad er avslått i vedtak datert ${vedtaksdato.toReadable()}."
     }
 
     fun beskrivelseRevurderingInnvilget(vedtaksPeriode: Pair<LocalDate, LocalDate>, gjeldendeVedtak: Vedtaksperiode): String {
-        return "Overgangsstønad revurdert pga endring i aktivitet i perioden ${vedtaksPeriode.vedtaksPeriodeToString()}" +
-                "Aktivitet: ${gjeldendeVedtak.aktivitet.name.enumToReadable()}"
+        return "Overgangsstønad revurdert. Periode ${vedtaksPeriode.vedtaksPeriodeToString()}." +
```

Comment:
Burde kanskje ha en space etter punktum her? 
```suggestion
        return "Overgangsstønad revurdert. Periode ${vedtaksPeriode.vedtaksPeriodeToString()}. " +
```